libs/markets/src/lib/components/markets-data/markets-data.tsx

```diff
@@ -0,0 +1,21 @@
+import { useMarketNamesMap } from '../../hooks/use-market-names-map';
+import { useMarketsMap } from '../../hooks/use-markets-map';
+import { useDataProvider } from '@vegaprotocol/data-provider';
+import { marketsProvider } from '../../markets-provider';
+
+export const MarketsData = () => {
```

Comment:
I'd also add that I think we should just keep MarketsData and the useMarketsMap hook in the trading app. And not add it to libs yet